proposals/ssss-distributed-actor-runtime.md

```diff
@@ -1179,13 +1270,12 @@ We see that as we decode our wire envelope, we are able to get the header sectio
 Next, we need to prepare for the decoding of the message section. This is done by implementing the remaining protocol requirements on the `ClusterTargetInvocation` type we defined earlier, as well as implementing a decoding iterator of type `DistributedTargetInvocationArgumentDecoder`, as shown below:
 
 ```swift
-extension ClusterSystem.ClusterTargetInvocation { // TODO: Split it into 2 separate "sides"
-    
-  // === Receiving / decoding -------------------------------------------------
-  
-  typealias ArgumentDecoder = ClusterTargetArgumentDecoder
-  
-  // FIXME: would like the visitor here too to avoid the array...
+final class ClusterTargetInvocationDecoder: DistributedTargetInvocationDecoder {
```

Comment:
Maybe we shouldn't advertise `final` just yet since there are issues in implementation?

Reply:
haha right let's not :)